tests/make.zunit

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env zunit
+
+@setup {
+    load ../fzf-zsh-completions.plugin.zsh
+    TEMP=$(mktemp -d)
+    pushd $TEMP
+}
+
+@test 'Testing completion: make **' {
+    MAKEFILE_CONTENT=$(cat <<< "
```

Comment:
Can you just put this content out of the test file and move to `tests/_support` as `npm` does?

Reply:
OK. It will fix it.